### PR TITLE
Change CSS to avoid Jasmine bug

### PR DESCRIPTION
**Pull request by prather-mcs**
I use pictures to show how this bug makes the project more confusing
to students here:
http://discussions.udacity.com/t/found-an-actual-bug-in-jasmine/17025

I reported and fixed the Jasmine bug here:
https://github.com/jasmine/jasmine/issues/844

[Review on Reviewable](https://reviewable.io/reviews/udacity/frontend-nanodegree-feedreader/9)

### CODE REVIEW

**Comment by walesmd** [comment]
@prather-mcs Can you resolve the current conflicts? If so, I'll gladly merge this.

**Comment by prather-mcs** [comment]
Hello @walesmd.

Did I resolve current conflicts? And keep the CSS changes I intended?

If so, all thanks is to [your Git & GitHub courseware](https://www.udacity.com/wiki/ud775), which I used alongside the normal Git documentation to get the PR up-to-date and ready to merge.

File: index.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
             <h1 class="header-title">Feeds</h1>
         </div>
 
-        <div class="menu">
+        <div class="slide-menu hidden">
```
Comment (walesmd) [inline index.html]: Can you remove the `hidden` class from this element? That class does not exist in the CSS file and was removed in prior commits.

**Comment by prather-mcs** [comment]
My apologies for not quite merging it in the right way for this project. I was so focused on taking the right steps with git remotes, branches, and forks, that I failed to actually get the text content right at the merge conflict line.

With my latest change, that mistake should have been amended.

**Pull request merged**